### DISCUSSION TITLE
How to change the "Menu" and "Return to top" words.

### DISCUSSION THREAD

**Discussion by otabekoff** [vuejs/vitepress#1725] [Q&A]
How to change the "Menu" and "Return to top" words which are just below the navbar in some pages. I want to change them like docFooter object. Is it possible?

**Comment by xu4wang** [comment]
```https://github.com/vuejs/vitepress/blob/main/src/client/theme-default/components/VPLocalNav.vue```

did a quick search, and found the one you are looking for are hard coded in the above component come with the default theme.  So it's not easy to config it. I guess it will need to customize theme.
